### ISSUE TITLE
Registry - replace extension for all current users?

### ISSUE THREAD

**Issue by patlillis** [adobe/brackets#13512]
I have an extension "Show Whitespace VS", that is a fork of a different extension. I am in the process of merging my extension upstream to the "Show Whitespace" extension, and am wondering if there is a way to automatically replace my extension with the "Show Whitespace" extension for all current users (similar to the current upgrade mechanism, but it would "upgrade" to a different extension).

Thanks.

**Comment by swmitra** [comment]
@patlillis In it's current form we don't have such mechanism as extensions are identified by extension ID and in this case both are having different ID. So there isn't any logical way to correlate them. 

* One thing which you can try ( It's extremely hacky by nature ) is to upload the original extension with correct attribution for the actual extension in package.json, but use your extensions ID. In this case, Brackets would detect a possible upgrade, and if current users of your extension upgrade, they will eventually get the original extension. The problem is, this hack would work only once and beyond that you need to keep publishing the original extension every time it upgrades by wrapping it with your extension ID. 

To summarize, there is no official way to achieve what you are asking and I can't think of a sustainable solution.

**Comment by petetnt** [comment]
One way I could think of would be to push a new version and one time alert the users about moving onto another extension.
